diary-alexander-jaffray/modernized/15-notes-diary.adoc

```diff
@@ -19,47 +19,47 @@ on the retrospective form of this part of the Diary.
 This Robert Burnet was the father of Bishop Burnet,
```

Comment:
It won't let me put a note at the first `=== Note A. `but I think that these note headings would look better if they were all centered.  So lets center them all, like this:

```
[.centered]
=== Note A.
```

Reply:
I was going to mention that too!